samples/sample_hevc_fei/include/ref_list_manager.h

```diff
@@ -117,6 +107,21 @@ class EncodeOrderControl
         m_encoding.resize(0);
 
         m_lastFieldInfo.Reset();
+        m_lastIDR = 0;
+        m_frameOrder = 0;
+
+        m_lastTask.Reset();
+        // make m_lastTask invalid:
+        Fill(static_cast<HevcDpbFrame & >(m_lastTask), IDX_INVALID);
```

Comment:
shouldn't all the code below be part of task.Reset()?

Reply:
Looks like it should. I'll check the code where Reset is called and submit the change if there are no conflicts with this change.